composer.json

```diff
@@ -39,35 +39,34 @@
     "psr/log": "^3.0.0",
     "sentry/sentry-symfony": "^4.2",
     "swagger-api/swagger-ui": "^5.1",
-    "symfony/amazon-mailer": "^6.4.3",
+    "symfony/amazon-mailer": "^7.0.0",
```

Comment:
These should all flip back to `@stable` to revert #5259

Reply:
handled. :heavy_check_mark: